akvo/rsr/tests/results_framework/test_results_framework.py

```diff
@@ -320,6 +330,79 @@ def test_period_update_does_not_create_deleted_periods(self):
         # Then
         self.assertEqual(1, child_indicator.periods.count())
 
+    def test_new_dimension_cloned_to_child(self):
+        """Test that new dimensions are cloned in children that have imported results."""
+        # Given
+        # # Child project has already imported results from parent.
+        indicator = self.indicator
+
+        # When
+        IndicatorDimension.objects.create(
+            indicator=indicator,
+            name='Baz',
+            value='Quux',
+        )
+
+        # Then
+        self.assertEqual(
+            IndicatorDimension.objects.filter(indicator=indicator).count(),
+            IndicatorDimension.objects.filter(indicator__in=indicator.child_indicators.all()).count(),
+        )
+
```

Comment:
I'd may be add a check to verify that the `parent_dimension` has been set correctly.

Reply:
I'll make the minor change myself.